src/tensor_ops/utilities/compatibility.cuh

```diff
@@ -14,7 +14,7 @@
 // }
 // #endif
 
-#if __CUDA_ARCH__ < 800
+#if __CUDA_ARCH__ < 700
```

Comment:
Can you change this to be the same check from this PR? https://github.com/huggingface/candle/commit/4a95d34c83453085c0d9dcecb19df9566024025e

which checks for cuda version < 12.2

```suggestion
#if (__CUDACC_VER_MAJOR__ < 12 || __CUDACC_VER_MINOR__ < 2) && __CUDA_ARCH__ < 800
```

Reply:
Sure thing, excellent idea.